package.json

```diff
@@ -2,20 +2,27 @@
   "name": "ilp-logger",
   "version": "1.1.2",
   "description": "Logging utility for Interledger modules",
-  "main": "src/index.js",
+  "main": "dist/lib/index.js",
+  "types": "dist/lib/index.d.ts",
   "files": [
-    "src/**/*.js",
-    "src/**/*.js.map",
-    "src/**/*.d.ts"
+    "src/lib/**/*.ts",
+    "dist/lib/**/*.js",
+    "dist/lib/**/*.js.map",
+    "dist/lib/**/*.d.ts",
+    "dist/doc"
   ],
   "scripts": {
-    "prepare": "tsc",
+    "clean": "rm -Rf .nyc_output && rm -Rf coverage && rm -Rf dist ",
     "build": "tsc",
     "lint": "tslint --project .",
-    "lint:fix": "tslint --fix --project .",
-    "test": "nyc mocha",
-    "doc": "typedoc --options typedoc.js src/index.ts",
-    "codecov": "codecov"
+    "pretest": "tsc --build",
+    "test": "nyc mocha --ui mocha-typescript --require source-map-support/register dist/test",
+    "doc": "typedoc --options typedoc.js src/lib/index.ts",
+    "codecov": "codecov",
+    "preversion": "npm run lint && npm test",
+    "version": "git add -A src",
```

Comment:
Why is version/postversion running git commands?

Reply:
@adrianhopebailie can you address this comment?

Reply:
`npm version` already runs git commands. It creates a new commit that includes the update to the `package.json`.

This ensures that working copies are also included so that if the tests etc are successful then there's no chance they break after version is tagged.

Happy to drop it if there is a concern that it's an unexpected side-effect

Reply:
@sentientwaffle @traviscrist 

I'm keen to merge this but wanted your okay first on this. Are you happy with my explanation or more comfortable if I take out the additional `npm version` behaviour?

Reply:
This seems like very unexpected and bad behavior. The user should definitely add or stash any changes explicitly. It should not happen magically in the background.

`npm version` makes a very specific and small commit that only changes one field

Reply:
I would take out additional npm version behavior.